src/Microsoft.TestPlatform.CommunicationUtilities/Messages/VersionedMessage.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities
+{
+    /// <summary>
+    /// New construct with version used for communication
```

Comment:
`New` becomes old ;) Better to not keep a sense of time in documentation. Add a remark specifying which version of vstest this was introduced and that this is the default message for protocol 2+.